### ISSUE TITLE
Looks like Insteon is dead

### ISSUE THREAD

**Issue by ellucianBret** [kuestess/homebridge-platform-insteonlocal#252]
Attempts to log in to their mobile app fail. Every page at insteon.com shows a vague message that they are gone.

Bret

**Comment by kuestess** [comment]
@ellucianBret Yep, they died about a month ago.  Plugin should still work fine.
